### PR TITLE
Update API examples using the DELETE /agents endpoint

### PR DESCRIPTION
**Pull request by mcarmona99**
Hi team,

This PR is related to [#6829](https://github.com/wazuh/wazuh/pull/6829).

In that PR, I have added the option `all` to the `status` variable enumerate used in the `DELETE /agents` endpoint. Adding to this, the variable `status` is now mandatory so an agent cannot be removed if the variable is left blank by accident.

In this PR, I have updated all the API examples using this endpoint. I have also updated the equivalence table.

Regards,
Manuel.

### CODE REVIEW

File: source/user-manual/api/equivalence.rst [hunk 1]
```diff
@@ -111,7 +111,9 @@ The following tables contain the equivalencies between old API 3.x and API 4.1 e
...
 |                                                  |                                                               |                                                                                                                                     |                                                                                                                                                                                                |
 |                                                  |                                                               |                                                                                                                                     | If no ``agents_list`` is provided no agents will be removed. In order to remove all agents, it must be specified with the ``all`` keyword.                                                     |
```
Comment (AdriiiPRodri) [inline source/user-manual/api/equivalence.rst:118]: For **agents_list** and **status** keyword

**Review by AdriiiPRodri — CHANGES_REQUESTED** [review]

File: source/user-manual/api/equivalence.rst (hunk 1 above)
Comment (mcarmona99) [reply source/user-manual/api/equivalence.rst]: Changed

**Review by AdriiiPRodri — APPROVED** [review]
LGTM!

**Review by davidfmiranda — APPROVED** [review]
LGTM!

**Pull request merged**